evm/loomevm.go

```diff
@@ -230,6 +201,7 @@ func (lvm LoomVm) Call(caller, addr loom.Address, input []byte, value *loom.BigU
 			events = lvm.receiptHandler.GetEventsFromLogs(
 				levm.sdb.Logs(), lvm.state.Block().Height, caller, addr, input,
 			)
+			levm.sdb.ResetLogs()
```

Comment:
Logs get clear every `State.Commit` but since now state is shared by many txs, we have to clear ou logs or we will end up having duplicates logs if there are multiple txs in one block.